requirements_tests.txt

```diff
@@ -1,15 +0,0 @@
--r requirements.txt
```

Comment:
Renamed to a more common name and removed the pip-ism syntax to make it package manager agnostic.